composer.json

```diff
@@ -71,7 +71,8 @@
     "phpstan": "phpstan analyse --ansi",
     "phpstan:gitlab-ci": "phpstan analyse --ansi --no-interaction --no-progress --error-format=gitlab > phpstan-report.json",
     "tests": "phpunit",
-    "tests:coverage:gitlab-ci": "phpunit --colors=never --coverage-text --coverage-cobertura .coverage/cobertura.xml --log-junit .coverage/junit.xml"
+    "tests:coverage:gitlab-ci": "phpunit --colors=never --coverage-text --coverage-cobertura .coverage/cobertura.xml --log-junit .coverage/junit.xml",
+    "yaml:lint": "yaml-lint config tests/Fixtures/Config"
```

Comment:
Where does this `yaml-lint` come from? Can I run it locally?

Reply:
Aah, it's this one, right? https://symfony.com/blog/new-in-symfony-5-1-misc-improvements-part-2#yaml-linter-command

Very nice, thx!

Reply:
As long as the yaml component is required by composer, it should be available :)